awx/ui_next/src/components/Lookup/Lookup.jsx

```diff
@@ -33,15 +33,6 @@ const SearchButton = styled(Button)`
 `;
 SearchButton.displayName = 'SearchButton';
 
-const InputGroup = styled(PFInputGroup)`
-  ${props =>
-    props.multiple &&
-    `
-    --pf-c-form-control--Height: 90px;
-    overflow-y: auto;
-  `}
-`;
-
 const ChipHolder = styled.div`
```

Comment:
this needs to be removed as well

Reply:
This variable ` --pf-c-form-control--Height: auto;` needs to kept.